oracle/scripts/start-worker.sh

```diff
@@ -8,12 +8,12 @@ LOGS_DIR="logs/"
 WORKER="${WORKERS_DIR}${1}.js"
 CONFIG="${2}.config.js"
 LOG="${LOGS_DIR}${2}.txt"
-TX_HASH="${3}"
+TX_HASH=${@:3}
```

Comment:
Imagine that we have dozens of unexecuted messages, it makes sense to specify them by a file rather than by a list of parameters.

Reply:
Will it be easy to make such file on a running oracle inside docker container. I assumed that confirm-relay script is usually executed via `docker-compose exec`/`docker-compose run`, so passing transaction hashes via command line seemed the easiest way for me.

We can also do something like:
```
docker-compose exec ... confirm-transfer $(cat ./tx_hashes.txt)
```

Reply:
Is there any limitation to pass data through the command line to avoid receiving `Argument list too long` by using the approach you suggested? Specifying the transactions hashes by a file will not meet such issue at all.

Reply:
Ok, seems that it is possible to accept both, direct hashes and files path at the same time in the CLI. I have updated the code in that way.